src/screens/Home.js

```diff
@@ -29,18 +31,36 @@ export default ({
   navigation,
   //state props
   categories,
+  learntPhrases,
   nativeLanguage,
   //actions
+  setPhrases,
   setCategories,
   setCurrentCategory,
-  setPhrases,
+  setLearntCategories,
+  setCurrentLearntCategory,
 }) => {
+  // const saveData = async list => {
+  //   try {
+  //     await AsyncStorage.setItem('categories', list);
+  //   } catch (error) {
+  //     alert('Error is seen !');
+  //   }
+  // };
+
   useEffect(() => {
     // fetch categories
     const categories = getAllCategories();
     setCategories(categories);
+    // saveData(categories);
+    // const learntcategories = getAllLearntCategories();
+    // setLearntCategories(learntcategories);
+    setLearntCategories(learntPhrases);
```

Comment:
setLearntCategories(learntPhrases);

action says "Categories" but you are passing "Phrases"